examples/expect-extend/package.json

```diff
@@ -3,13 +3,13 @@
   "version": "0.0.0",
   "name": "example-expect-extend",
   "devDependencies": {
-    "@babel/core": "*",
-    "@babel/preset-env": "*",
-    "@babel/preset-typescript": "*",
-    "@jest/globals": "workspace:*",
```

Comment:
@SimenB I changed `workspace:*` to `*`, bec it's more consistent between examples dir. but It resolved to jest version 27. I think using "*" to reference internal package instead of "workspace:*" is bad.

Reply:
should we move `*` from examples directory to `workspace:*`?

Reply:
yeah, let's

Reply:
let's what? lol.

Reply:
let's use `workspace:*` for all workspace deps 😀